yesod-form/yesod-form.cabal

```diff
@@ -45,6 +44,9 @@ library
     if flag(network-uri)
       build-depends: network-uri >= 2.6
 
+    if impl(ghc < 8.0)
```

Comment:
Since we already have a `base >= 4.10` dep, we only support this GHC anyway, so I think we can drop this conditional entirely.

Reply:
Indeed. Updated!